Status enum does not contain "NOT_FOUND".

#### Environment details

Concerning the status `Status` Enum
Found on macOS 10.14.6 with `"@googlemaps/google-maps-services-js@2.6.0"` installed via yarn. Typescript version 3.8.3.

#### Steps to reproduce

1. Import `Status` and try and use `Status.NOT_FOUND`. Typescript will complain.
2. Documentation [here](https://developers.google.com/places/web-service/details#PlaceDetailsStatusCodes) suggests it is a valid response status.

#### Code example

```ts
import {Client, Status} from '@googlemaps/google-maps-services-js';
const c = new Client({});
response = await c.PlaceDetails({
  params: {
    place_id: 'xyz',
    fields: [/* ... */],
    key: 'foobar',
  }
});

switch (response.data.status) {
  // ...
  case Status.NOT_FOUND:
    console.log("NOT FOUND");
    break;
  // ...
}

```

Will result in a typescript error

```
foobar:x:y - error TS2339: Property 'NOT_FOUND' does not exist on type 'typeof Status'.

xxx     case Status.NOT_FOUND:
```

--- a/src/common.ts
+++ b/src/common.ts
@@ -85,6 +85,8 @@ export enum Status {
   UNKNOWN_ERROR = "UNKNOWN_ERROR",
   /** indicates that the request was successful but returned no results. */
   ZERO_RESULTS = "ZERO_RESULTS",
+  /** indicates that the referenced location (place_id) was not found in the Places database. */
+  NOT_FOUND = "NOT_FOUND",
 }
 
 export interface PlacePhoto {

